src/nft/ZoraCreator1155Impl.sol

```diff
@@ -727,10 +727,6 @@ contract ZoraCreator1155Impl is
     /* start eip712 functionality */
     mapping(uint32 => uint256) public delegatedTokenId;
```

Comment:
should we put this in a `PremintStorage` contract?

Reply:
yes to be consistent that makes sense. It's out of the scope of this PR, but can do that as a separate PR.

Reply:
sounds good